Hallowspeak/Pages/Contact.razor

```diff
@@ -135,6 +149,10 @@
                 Message = string.Empty;
                 InvokeAsync(StateHasChanged);
             });
+
+            IpBlackList.Add(clientIp);
+
+            Task.Run(async () => { await Task.Delay(new TimeSpan(0, 1, 0)); IpBlackList.Remove(clientIp); });
```

Comment:
Have you tested that if you close/leave the page this code still runs? I'm like 80% sure that it won't x)